provisionssmsessionmanager_policy.tf

```diff
@@ -0,0 +1,53 @@
+# ------------------------------------------------------------------------------
+# Create the IAM policy that allows all of the permissions necessary to
+# provision the SSM Document resource and set up SSM session logging in this
+# account.
+# ------------------------------------------------------------------------------
+
+data "aws_iam_policy_document" "provisionssmsessionmanager_policy_doc" {
```

Comment:
Unrelated to this PR but this is replicated in a lot of places across our repositories. Is there a good way for us to de-duplicate this?

Reply:
The [cisagov/session-manager-tf-module](https://github.com/cisagov/session-manager-tf-module) could be modified to handle these roles and policies for us.

Reply:
I created https://github.com/cisagov/cool-system/issues/237 to document this.  @mcdonnnj please let me know if you want anything else here or if this conversation can be resolved.

Reply:
Nope, I just hadn't had a chance to look over the module and policy to create an issue. Thanks for taking care of that!